types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,15 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
+export const LOCATION_CHANGE: '@@router/LOCATION_CHANGE';
```

Comment:
Declaring const with value would be right choice if that constant exists at runtime in the module. Otherwise it would be incorrect representation. If the constant isnt present at runtime you want to declare it as a type and not constant.